File: PYTHON/Questões Uri/1129.py
control = 1
while control == 1:
    qtd = int(input())
    if qtd == 0:
        control = 0
        break
    for i in range(qtd):
        a,b,c,d,e=map(int, input().split(" "))
        if(a >=0 and a <= 127 and b > 127 and c > 127 and d > 127 and e > 127):
            print('A')
        elif(b >=0 and b <= 127 and a > 127 and c > 127 and d > 127 and e > 127):
            print('B')
        elif(c >=0 and c <= 127 and b > 127 and a > 127 and d > 127 and e > 127):
            print('C')
        elif(d >=0 and d <= 127 and b > 127 and c > 127 and a > 127 and e > 127):
            print('D')
        elif(e >=0 and e <= 127 and b > 127 and c > 127 and d > 127 and a > 127):
            print('E')
        else:
            print('*')

File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/1129.js
let control = 1;
while(control == 1){
    let qtd = +lines.shift()
    if(qtd == 0){
        control = 0;
        break;
    }
    for(let i = 0; i < qtd; i++){
        let[a,b,c,d,e] = lines.shift().split(" ").map(a => +a)
        if(a >=0 && a <= 127 && b > 127 && c > 127 && d > 127 && e > 127){
            console.log('A')
        }else if(b >=0 && b <= 127 && a > 127 && c > 127 && d > 127 && e > 127){
            console.log('B')
        }else if(c >=0 && c <= 127 && b > 127 && a > 127 && d > 127 && e > 127){
            console.log('C')
        }else if(d >=0 && d <= 127 && b > 127 && c > 127 && a > 127 && e > 127){
            console.log('D')
        }else if(e >=0 && e <= 127 && b > 127 && c > 127 && d > 127 && a > 127){
            console.log('E')
        }else{
            console.log('*')
        }
    }
}
